man/systemd-sbsign.xml

```diff
@@ -49,6 +49,16 @@
         <xi:include href="version-info.xml" xpointer="v257"/>
         </listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><option>validate-key</option></term>
+
+        <listitem><para>Checks that we can load the private key specified with
+        <option>--private-key=</option>.</para>
```

Comment:
The intended use case (using the appropriate env vars resulting in the pin being cached) should be documented here